students/YingGuo/session04/mailroom2.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+#session04, mailroom2
+
+from collections import defaultdict
+data = defaultdict(list)
+Sample = {"Tom":[100,200], "Tim":[3000], "Ken": [888,888,888], "Ted":[8000,5000,500], "Jon":[1000,6000]}
+for key in Sample.keys():
+    data[key] = Sample[key]
```

Comment:
Wondering why you had to create `Sample` separately and copy like this? I suppose it's partly because we didn't talk about how to initialize `defaultdict(list)` with values. You could have simply done the following:
```
data = defaultdict(list, {"Tom":[100,200], "Tim":[3000], "Ken": [888,888,888], "Ted":[8000,5000,500], "Jon":[1000,6000]})
```
for lines 6-9...